gps_waypoint_follower_demo:
  ros__parameters:
    number_of_gps_waypoints: 21
    #long, lat, alt
    gps_waypoint0: [9.677703999088216e-07, -5.306676831178058e-05, 0.6442248001694679]
    gps_waypoint1: [9.677703999088216e-07, -5.306676831178058e-05, 0.6442248001694679]
    gps_waypoint2: [3.1139105755464975e-06, -0.00017456442820206204, 0.6444645449519157]
    gps_waypoint3: [4.22315917828845e-06, -0.00023834113403818424, 0.6447720658034086]
    gps_waypoint4: [5.733249480810798e-06, -0.00032208389604156264, 0.6448167748749256]
    gps_waypoint5: [7.383338580758601e-06, -0.00041311675834673534, 0.6448821565136313]
    gps_waypoint6: [9.214683482042934e-06, -0.0005185112536830417, 0.644977854564786]
    gps_waypoint7: [1.0501804114763628e-05, -0.0005927400620565918, 0.6450572097674012]
    gps_waypoint8: [4.169383611283205e-05, -0.0006143364570898212, 0.6346865268424153]
    gps_waypoint9: [9.319715737387455e-05, -0.000620772355007051, 0.6348643703386188]
    gps_waypoint10: [0.00016136807000342732, -0.000630472090460399, 0.6346333334222436]
    gps_waypoint11: [0.00019289536480012745, -0.0006025216750314517, 0.6347960121929646]
    gps_waypoint12: [0.00016725512736177336, -0.0005682180030561106, 0.6347330678254366]
    gps_waypoint13: [9.548175925016343e-05, -0.0005566846481087024, 0.6348178731277585]
    gps_waypoint14: [2.1370495894155417e-05, -0.0005427454926971877, 0.6447623465210199]
    gps_waypoint15: [1.4067128652996008e-06, -0.0004089532192350932, 0.6448766821995378]
    gps_waypoint16: [1.251103351683063e-05, -0.000262679057585972, 0.6447812123224139]
    gps_waypoint17: [1.5709760573368857e-05, -0.00018905817925624448, 0.644750733859837]
    gps_waypoint18: [1.4568988705645404e-05, -0.0001429542567161331, 0.6447363374754786]
    gps_waypoint19: [1.2338132128329487e-05, -9.000069320371602e-05, 0.6447243737056851]
    gps_waypoint20: [8.37498018946476e-06, -2.402470336058297e-05, 0.6447164406999946]

